src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -279,14 +295,14 @@ private static ISymbol TryGetSymbol(NameSyntax name, SymbolInfo symbolInfo, Sema
             return false;
         }
 
-        private bool IsSymbolCalledVar(ISymbol symbol)
+        private bool IsSymbolWithName(ISymbol symbol, string name)
         {
             if (symbol is INamedTypeSymbol namedType)
             {
-                return namedType.Arity == 0 && symbol.Name == "var";
+                return namedType.Arity == 0 && symbol.Name == name;
             }
 
-            return symbol != null && symbol.Name == "var";
+            return symbol != null && symbol.Name == name;
```

Comment:
💭 Seems like the type/arity check could come last since the name check is the most likely to fail here.